data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -1065,7 +1065,7 @@
 
                 [message]
                     speaker=Tallin
-                    message= _ "I don’t know, see if it talks."
+                    message= _ "I don’t know. See if it talks."
```

Comment:
Both feel the same to me, so I'd prefer "no change".

Reply:
Yea this is unnecessary.